h5grove/tornado_utils.py

```diff
@@ -67,7 +67,7 @@ def prepare(self):
 
     def write_error(self, status_code: int, **kwargs: Any) -> None:
         self.prepare()
-        return super().write_error(status_code, **kwargs)
+        self.finish({"message": self._reason})
```

Comment:
...and rethrow a `HTTPError` for tornado to be able to plug it in the process calling `write_error`.